classes/php.php

```diff
@@ -41,6 +41,11 @@ public function setBinaryPath($phpPath = null)
 			}
 		}
 
+		if (defined('HHVM_VERSION'))
```

Comment:
after thinking a bit more about this patch, I came up to the conclusion that it imcould be wrong. 

Let me explain why: using the constant to find wether we run on hhvm or php can lead to wrong behavior as the user can run atoum (the runner) on hhvm but ask him to run the tests (the executoe)  on php via the ˋ-p` flag. using the constant.

I'm working on a clearer solution right now and hopefully it will be ready soin.